front/app/containers/Admin/formBuilder/components/FormBuilderToolbox/ToolboxItem.tsx

```diff
@@ -49,10 +49,10 @@ const ToolboxItem = ({ icon, label, onClick }: Props) => {
         fill={colors.adminTextColor}
         width="20px"
         height="20px"
-        marginLeft="12px"
+        marginLeft="16px"
         name={icon}
       />
-      <Text fontSize="s" marginLeft="8px" color="text">
+      <Text fontSize="s" ml="8px" mt="13px" mb="12px" color="text">
```

Comment:
Sorry - just noticed the 13px here for marginTop. Would 12px work (to keep it a multiple of 4)? :) Then we could use `my="12px"`

Reply:
The extra pixel makes the icon, text and '+' icon align vertically so I think we should leave it if that's ok?

Reply:
Ok makes sense! I'll just check with Iva quickly :) We've been pretty strict with that lately, so I just want to be sure it's alright.

Reply:
I've pushed a change that completely tidies-up the toolBox item margin/padding - now all the white space comes from the `Box` padding of 18px...loads neater!

I've also added a `role="button"` for accessibility and tweaked the hover cursor to a pointer (as opposed to move) as it's a clickable element. We can tweak that when we implement drag and drop.

Reply:
Sounds great - thanks Ben!